lib/lndclient/LndClient.ts

```diff
@@ -1083,15 +1085,15 @@ class LndClient extends SwapClient {
     channels.forEach((channel) => {
       if (channel.getRemotePubkey() === remoteIdentifier) {
         const [fundingTxId, outputIndex] = channel.getChannelPoint().split(':');
-        const closePromise = this.closeChannelSync(fundingTxId, Number(outputIndex), force);
+        const closePromise = this.closeChannelSync(fundingTxId, Number(outputIndex), force, fee);
         closePromises.push(closePromise);
       }
     });
     await Promise.all(closePromises);
   }
 
   /** A synchronous helper method for the closeChannel call */
-  public closeChannelSync = (fundingTxId: string, outputIndex: number, force: boolean): Promise<void> => {
+  public closeChannelSync = (fundingTxId: string, outputIndex: number, force: boolean, fee: number): Promise<void> => {
```

Comment:
Minor but I think `fee` can/should still be optional here by just assigning `fee = 0`.